### DISCUSSION TITLE
Update & Consistent Showcase screenshots

### DISCUSSION THREAD

**Discussion by hasezoey** [tramhao/termusic#285] [Ideas]
The Current showcase screenshots are somewhat out-of-date, and i wanted to propose that we maybe have a script to generate screenshots consistently, via somethings like [`vhs`](https://github.com/charmbracelet/vhs).

For example [`ratatui`](https://github.com/ratatui-org/ratatui/tree/main/examples) also uses `vhs` to generate screenshots for their examples.

**Comment by tramhao** [comment]
That is awesome!
